js/ui/shellDBus.js

```diff
@@ -562,6 +570,10 @@ const AppStoreService = new Lang.Class({
         // since the replace operation degenerates to
         // append if the source icon was not available
         IconGridLayout.layout.replaceIcon(originalId, replacementId, IconGridLayout.DESKTOP_GRID_ID);
+
+        if (!iconWasVisible) {
+            _reportAppAddedMetric(replacementId);
```

Comment:
Nit: no need for braces in single-line if statements

Reply:
Fixed in a followup force push